controllers/operands/kubevirt.go

```diff
@@ -880,8 +878,19 @@ func getMandatoryKvFeatureGates(isKVMEmulation bool) []string {
 // get list of feature gates or KV FG list
 func getKvFeatureGateList(fgs *hcov1beta1.HyperConvergedFeatureGates) []string {
 	checks := getFeatureGateChecks(fgs)
+
 	res := make([]string, 0, len(checks)+len(mandatoryKvFeatureGates)+1)
-	res = append(res, mandatoryKvFeatureGates...)
+
+	if fgs.DownwardMetrics != nil && !*fgs.DownwardMetrics {
```

Comment:
please implement this logic in `getFeatureGateChecks` and keep this as it is.

Reply:
This change will force us to change some tests behavior which relies on the `hardCodeKvFgs`. Would you be fine with such change? Just follow the `hardCodeKvFgs` usages, it is widely used in the codebase. I implemented in this way because It creates minimal changes to the test codebase.